kit/app/views/current/r1/603-5829-4.html

```diff
@@ -80,12 +92,12 @@ <h1 class="govuk-heading-xl">
         This qualification is approved by the Department for Education as full and relevant, subject to the following notes:
 
         <ul class="govuk-list govuk-list--bullet">
-          <li>The course must be assessed within the EYFS in an Early Years setting in England. </li>
+          <li>The course must be assessed within the early years foundation stage (EYFS) in an early years setting in England. </li>
         </ul>
       </p>
 
       <p>
-        This is a level 3 qualification which was started before 1 September 2014. Someone who holds this qualification can be included in staff:child ratios at level 3, as long as they meet all other requirements as set out in the <a href="https://www.gov.uk/government/publications/early-years-foundation-stage-framework--2">Early Years Foundation Stage</a>.
+        This is a level 3 qualification which was started after 1 September 2014. Someone who holds this qualification can be included in staff:child ratios at level 3, as long as they meet all other requirements as set out in the <a href="https://www.gov.uk/government/publications/early-years-foundation-stage-framework--2">Early years foundation stage statutory framework (EYFS)</a> and the <a href="https://www.gov.uk/government/publications/early-years-qualification-requirements-and-standards">Early years qualification requirements and standards</a> document.
```

Comment:
Does the word 'Early' after the link need to be lowercase too?

Reply:
```suggestion
        This is a level 3 qualification which was started after 1 September 2014. Someone who holds this qualification can be included in staff:child ratios at level 3, as long as they meet all other requirements as set out in the <a href="https://www.gov.uk/government/publications/early-years-foundation-stage-framework--2">early years foundation stage statutory framework (EYFS)</a> and the <a href="https://www.gov.uk/government/publications/early-years-qualification-requirements-and-standards">early years qualification requirements and standards</a> document.
```